lib/Parser/Parse.cpp

```diff
@@ -5650,6 +5652,8 @@ bool Parser::ParseFncDeclHelper(ParseNodeFnc * pnodeFnc, LPCOLESTR pNameHint, us
             CHAKRATEL_LANGSTATS_INC_LANGFEATURECOUNT(ES6, StrictModeFunction, m_scriptContext);
         }
 
+        ProcessCapturedNames(pnodeFnc);
```

Comment:
we don;t do this for global function, right?

Reply:
Yes, we don't do this for global functions.